demo/mobile/src/app/sync/syncGlobalNetworkHandler.js

```diff
@@ -2,7 +2,11 @@ module.exports = function createHandler(cloudURL, cloudPath, $http) {
   var handler = function(params, success, failure) {
     var url = cloudURL + cloudPath + params.dataset_id;
     var payload = params.req;
-    $http.post(url, payload, {}).then(success).catch(failure);
+    $http.post(url, payload, {}).then(function(response) {
```

Comment:
I think it might be important to not pass an empty config object as the third param, it might override some important global defaults like caching or headers in `$httpProvider.defaults`, but I'm not completely sure.

Angularjs docs say it's an optional param.